src/coreComponents/managers/initialization.cpp

```diff
@@ -110,10 +110,14 @@ void addUmpireHighWaterMarks()
     // This is a little redundant since
     std::size_t const mark = rm.getAllocator( allocatorName ).getHighWatermark();
     std::size_t const totalMark = MpiWrapper::sum( mark );
-    GEOSX_LOG_RANK_0( "Umpire " << std::setw( 15 ) << allocatorName << " high water mark: " <<
+    std::size_t const maxMark = MpiWrapper::max( mark );
+    GEOSX_LOG_RANK_0( "Umpire " << std::setw( 15 ) << allocatorName << " total high water mark: " <<
                       std::setw( 9 ) << LvArray::system::calculateSize( totalMark ) );
+    GEOSX_LOG_RANK_0( "Umpire " << std::setw( 15 ) << allocatorName << " max high water mark: " <<
+                      std::setw( 9 ) << LvArray::system::calculateSize( maxMark ) );
 
-    pushStatsIntoAdiak( allocatorName + " high water mark", mark );
+    pushStatsIntoAdiak( allocatorName + " total high water mark", mark );
+    pushStatsIntoAdiak( allocatorName + " max high water mark", mark );
```

Comment:
I can remove this or restructure it to make the output a bit cleaner, it is useful in the packing scaling since the max pinned memory directly correlates to our packing data.

Reply:
Maybe rename it to like "sum across ranks" and "rank max" or something.